ansible/roles/openshift_setup/tasks/main.yml

```diff
@@ -128,15 +127,6 @@
       - "{{ docker_images_group1 }}"
       - "{{ docker_images_group2 }}"
 
-  - name: Remove {{ oc_host_config_dir }}/master/master-config.yaml when resetting cluster
```

Comment:
The original context was catasb only worked on ec2 provisioning.
reset was intended to reuse the provisioned instance, just reset the oc cluster portion.

Then we added in the local options, so reset became much closer to setup_local_environment, not sure off hand what the diffs are if there are any on local context.

Essentially we want reset to be capable of clearing out the prior config and resetting from scratch.